pyrefinebio/__init__.py

```diff
@@ -12,3 +12,9 @@
 from .sample import Sample
 from .token import Token
 from .transcriptome_index import TranscriptomeIndex
+
+from .exceptions import (
```

Comment:
Oh I just noticed this, but I prefer to stick to absolute rather than relative imports. The biggest reason is that if we ever wanted to move a file, we'd have to change all the imports as well with relative imports.